discussions/index.md

```diff
@@ -2,4 +2,5 @@
 
 > What is the purpose of this project?
 
-There should be a answer...
+> This project is for digitalizing small businesses which still use paper for saving their data.
```

Comment:
- Don't forget to create PR
- If you are answering on questions, then please don't use ">" symbol at the beginning of your answer

Reply:
Please, checkout https://www.markdownguide.org/
There are some hints and best practises as well:)